### PR TITLE
Admin setting to disable default follows

### PR DESCRIPTION
**Pull request by ashpieboop**
I wanted to disable auto follows on my instance, so I implemented a setting.

### CODE REVIEW

File: app/services/bootstrap_timeline_service.rb [hunk 1]
```diff
@@ -28,7 +28,13 @@ def autofollow_bootstrap_timeline_accounts!
   def bootstrap_timeline_accounts
     return @bootstrap_timeline_accounts if defined?(@bootstrap_timeline_accounts)
 
-    @bootstrap_timeline_accounts = bootstrap_timeline_accounts_usernames.empty? ? admin_accounts : local_unlocked_accounts(bootstrap_timeline_accounts_usernames)
+    accounts = []
+
+    if Setting.enable_bootstrap_timeline_accounts || Setting.enable_bootstrap_timeline_accounts.nil?
```
Comment (ArisuOngaku) [inline app/services/bootstrap_timeline_service.rb]: Please particularly pay attention to this line; I tried to make it enabled by default, is this good?

**Comment by ArisuOngaku** [comment]
Ok I think circleci isn't triggering because I actually set it up on the PR's repo, hold on I'm creating another one

File: app/services/bootstrap_timeline_service.rb (hunk 1 above)
Comment (ThibG) [reply app/services/bootstrap_timeline_service.rb]: Default values are set in `config/settings.yml`. Furthermore, I wouldn't edit the `bootstrap_timeline_accounts` method, but simply prevent `autofollow_bootstrap_timeline_accounts!` from running unless `Setting.enable_bootstrap_timeline_accounts` is true.

**Review by ClearlyClaire — CHANGES_REQUESTED** [review]

**Comment by ArisuOngaku** [comment]
@ThibG thanks for your review and for teaching me about config/settings.yml. I think you're right, this is a way better solution and I should have looked this through.

**Review by ClearlyClaire — APPROVED** [review]

**Pull request merged**